tests/integration/header/header_test.go

```diff
@@ -29,6 +32,17 @@ import (
 
 type testDispatcher struct{}
 
+func (testDispatcher) ContentType(resp safehttp.Response) (string, error) {
+	switch resp.(type) {
+	case safehtml.HTML, *template.Template:
+		return "text/html; charset=utf-8", nil
+	case safehttp.JSONResponse:
+		return "application/json; charset=utf-8", nil
+	default:
+		return "", errors.New("not a safe response")
+	}
+}
+
```

Comment:
Please file an issue to carve out the `testDispatcher`, as it feels it should be the same for most of the tests.